clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/is_config_synchronized

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I'm not convinced that check_config_sync should return an error, when it successfully checked the config synchronisation status.